changelog/db.changelog-1.52.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Sam Plackett" id="1">
+    <tagDatabase tag="v1.52.0" />
+  </changeSet>
+  <changeSet author="Sam Plackett" id="2">
+    <insert tableName="holdCategories">
+      <column name="holdCategoryId" value="122" />
+      <column name="schemeId" value="12" />
+      <column name="name" value="Manual ledger hold" />
```

Comment:
Need to add a few additional hold categories.

1.DAX rejection
2. Awaiting debt enrichment
3. Ex-gratia
4. Hardship case
5. Non-payable
6. Withdrawal
7. Manual payment
8. Bridging payments
9. Other admin
10. Bank account anomaly
11. Recovery
12. Top up
13. Manual ledger hold

Reply:
additional hold categories added